### DISCUSSION TITLE
textbox does not support textvariable binding for customtkinder.StringVar?

### DISCUSSION THREAD

**Discussion by tswelectronics** [TomSchimansky/CustomTkinter#2527] [General]
Am I mistaken but it appears label supports textvariable= but textbox does not?

**Comment by dipeshSam** [comment]
In `Tkinter`/`customtkinter`, the `Textbox` is not only a container for textual data but can also contain different font information, images, and windows as a single character element. This is why `Textbox` can not simply accept a `textvariable`.

Regards.

**Reply by tswelectronics** [reply]
Thanks for the reply. I was sure tkinter.textbox supports textvariable

**Reply by tswelectronics** [reply]
No I think I am mistaken. Is it possible to create a read only text area with customTkinter that uses textvariable binding?
